src/server/api/user.js

```diff
@@ -33,12 +34,15 @@ export const resolvers = {
       if (!user || !user.id) {
         return []
       }
-      let orgs = r.table('user_organization')
-        .getAll(user.id, { index: 'user_id' })
+      let orgs = r.knex.select('organization.*')
```

Comment:
Can you add a method to get this info in:
src/server/models/cacheable-queries.js
which leverages cache data `texterinfo-${userId}` when available (and skips the db)?

Reply:
hey @schuyler1d - you want the info in this query to be cached (when possible and looked at if available)?

```
let orgs = r.knex.select('organization.*')
        .from('organization')
        .join('user_organization', 'organization.id', 'user_organization.organization_id')
        .where('user_organization.user_id', user.id)
```

Reply:
The information for this query will already be in the cache if it exists.  The query here is basically `userOrgsWithRole(role)`  We should implement this funciton in cacheable-queries/ so that
```
async function userOrgsWithRole(role) {
  if (r.redis) {
     const userKey = `texterinfo-${userId}`
     const userOrgs = await r.redis.hgetallAsync(userKey)
     if (userOrgs) {
        // below we return all keys with values equal to or higher than the requested role
       // turn the dict into an array and filter 
       return orgs
    }
  }
  let orgs = r.knex.select('id')
        .from('organization')
        .join('user_organization', 'organization.id', 'user_organization.organization_id')
        .where('user_organization.user_id', user.id)
  return orgs
}
```
Also note we only call that resolver asking for organization id, so I don't think we should bother loading any of the other data until we need it.

Reply:
I think we need the name as well @schuyler1d for the resolver.